### DISCUSSION TITLE
Profalux Zigbee intégration : help me with the procedure

### DISCUSSION THREAD

**Discussion by jerome6994** [Koenkk/zigbee2mqtt#7899] [Devices]
hello,

I am new to the subject but I am trying to integrate my Profalux shutters.
My english is not great and I use google translate. 

I was able to do the integration manipulation of my pane on zigbee2mqtt and I have the log in debug mode. 
But i don't know what to do with it and how to get the payload information to you and i didn't figure out how to fill the js file and where i need to put it on my system. 

Thank you for your help.

**Comment by meyblum** [comment]
> hello,
> 
> I am new to the subject but I am trying to integrate my Profalux shutters. My english is not great and I use google translate.
> 
> I was able to do the integration manipulation of my pane on zigbee2mqtt and I have the log in debug mode. But i don't know what to do with it and how to get the payload information to you and i didn't figure out how to fill the js file and where i need to put it on my system.
> 
> Thank you for your help.

Hello jerome6994,

Can you explain to me how did you integrate your Profalux shutter in HA with Z2M?
Did you find how to control them with HA?

Thank you!
